Lib/_pydatetime.py

```diff
@@ -2112,12 +2132,14 @@ def ctime(self):
             self._hour, self._minute, self._second,
             self._year)
 
-    def isoformat(self, sep='T', timespec='auto'):
+    def isoformat(self, sep='T', timespec='auto', basic=False):
         """Return the time formatted according to ISO.
 
         The full format looks like 'YYYY-MM-DD HH:MM:SS.mmmmmm'.
         By default, the fractional part is omitted if self.microsecond == 0.
 
+        If *basic* is true, separators ':' and '-' are removed from the output.
```

Comment:
```suggestion
        If *basic* is true, separators ':' and '-' are omitted from the output.
```

It sounds better to me, "removed" may be misleading.

Previous line:

> By default, the fractional part is omitted if self.microsecond == 0.